apps/client/src/components/molecules/input/Input.component.tsx

```diff
@@ -0,0 +1,46 @@
+import cs from 'classnames';
+import React, { ReactElement, useState } from 'react';
+
+export interface InputProps {
+  value?: string;
+  placeholder?: string;
+  props?: React.InputHTMLAttributes<HTMLInputElement>;
+  isValid?: boolean;
+  isInvalid?: boolean;
```

Comment:
Why do we have a `isValid` and a `isInvalid`. What does `isValid` do ? shall we rename it give the proper meaning

Reply:
In UI we have 3 states for input,

1. Initial/Default state (Mostly the empty state)
2. When the value is valid
3. When the value is invalid

![image](https://user-images.githubusercontent.com/33794760/217738982-a4060ccd-d7a1-4e58-a950-fcb49717c9d6.png)

Reply:
If no 2 states can exist at the same time, shall we use an enum to indicate the state?
enum InputState {
   Initial,
   Valid,
   Invalid,
}

Reply:
Changed 14eb6d9e00d4f5a8eb5c43963a9013f2f86db5c2